spacy/language.py

```diff
@@ -445,6 +445,10 @@ def update(self, docs, golds, drop=0.0, sgd=None, losses=None, component_cfg=Non
             if isinstance(doc, basestring_):
                 doc = self.make_doc(doc)
             if not isinstance(gold, GoldParse):
+                expected_keys = ("words", "tags", "heads", "deps", "entities", "cats")
+                unexpected_keys = [k for k in gold if k not in expected_keys]
+                if unexpected_keys:
+                    raise ValueError(Errors.E151.format(unexpected_keys=unexpected_keys))
```

Comment:
Lets add "links" to this list to ensure the upcoming entity linking functionality is already accounted for :-)